### ISSUE TITLE
PDF text editable

### ISSUE THREAD

**Issue by IosSos** [xhtml2pdf/xhtml2pdf#552]
hi,

how to do make a pdf with text editable on pdf? 
what is the tag that i can use?

**Comment by timoludwig** [comment]
What do you mean? Fillable form input fields? I think this is just not possible with this library and probably also out of scope for the future.

**Comment by IosSos** [comment]
this is an example: https://ibb.co/wrvyn2w

@timoludwig

**Comment by timoludwig** [comment]
Yes, as I said, this is not possible at the moment and probably also not planned for the future.

**Comment by wdixon** [comment]
I have a question regarding this forms topic.  I see that 

[Form Support](https://github.com/xhtml2pdf/xhtml2pdf/blob/ba89b5ea69dd743e514a2d4f6ea89c933f920ba5/xhtml2pdf/tags.py#L439-L491)

seems to be implemented, and by changing the 
`if 0:`
to
`if 1:`
the text input field seemed to work properly...  Is there a particular reason this is disabled?

**Comment by luisza** [comment]
I checked that, but `if 0:` is there because the code looks in progress and didn't work well.
Maybe in the next release I will code support for forms :) I interested on this functionality.
